### PR TITLE
Add contact form

### PR DESCRIPTION
**Pull request by Nyame-Wolf**
In this pull request I;
- Created a contact form
- Added validations for contact form elements and ensured that all fields have to be filled.
- Ensured that name field accepted a maximum of 30 characters and Text area a maximum of 500 characters.
- Ensured that all fields have valid types
- Ensured that all fields have placeholder text

#Reviewer Recommendations:
- Added button hover and active effect on get-in-touch button
- Fixed header margin, font-style and color to match figma design
- Added font-style, weight and font-size to input elements and texarea
- Fixed font sizes for font-awesome icons
- Fixed  forms background color to match figma design
- Changed licence name to MIT.md as recommended by reviewer.
- Modified readMe so that project description comes after project title

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,7 +14,7 @@
 
 ## Getting Started
 
-**This a set up of my mobile version of my portfolio project. It adds the works and about sections to the already created headline section. It uses a figma template 2 from microverse student portal..**
+**This a set up of my mobile version of my portfolio project. It adds a contact form to the already created headline, works and about sections. It uses a figma template 2 from microverse student portal..**
```
Comment (tresorsawasawa) [inline README.md:17]: - [ **OPTIONAL** ] What do you think if you put the project's description just after the project's title by starting with a title **Description** then details below the title as it is in the Microverse [README template](https://github.com/microverseinc/readme-template/blob/master/README.md). And if possible fulfill the **Getting Started** section with some instructions on how to set up your project locally. You can take a look at this [README](https://github.com/tresorsawasawa/MyPortfolio/edit/develop/README.md) as example.

**Review by tresorsawasawa — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @Nyame-Wolf 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 

## Rewquired changes :recycle: :recycle: 
- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 2]
```diff
@@ -424,3 +426,67 @@ h4 {
...
+}
+
+form button {
+  background-color: #ff6b00;
+  color: #fff;
+  border: 0 solid #fff;
+  margin-top: 2em;
+  padding: 1em;
+  margin-bottom: 4em;
+}
```
[the remark this answers is not in the document]
Comment (ishimwezachee) [reply style.css:496]: - [ ] Could you improve your font-weight as previous reviewer suggested, Try your best to resemble based on provided figma design. 

Your design
image
Figma design
image

**Review by zieeco — APPROVED** [review]
# status: APPROVED :green_circle: :100: 

Hi, @studentname, 


68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f78543078657a514755357843444a754350652f67697068792e676966


Good job so far 👍 Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights ⚡ 
✅ Nice design 🔥 
✅ Linters are passing 👏 


## Optional suggestions
- N/A


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**